website/blog/2024-08-12-release-0.75.md

```diff
@@ -0,0 +1,487 @@
+
+```tsx
+import {TouchableHighlight} from 'react-native';
+const ref1 =
+  useRef<React.ElementRef<typeof TouchableHighlight>>();
+// or
+const ref2 = useRef<View>();
+```
+
+#### Last version supporting minSdk 23 and minIOSVersion 13.4
+
+These are not breaking changes in 0.75 per-se, but we want to share how React Native 0.75 will be the last version of React Native to support minSdk 23 (Android 6.0) and minIOSVersion 13.4.
+
+Starting from React Native 0.76, the minSdk version will be 24 (Android 7.0) and the minIOSVersion will be 15.1.
+
+You can read more about it in our official announcement [for Android](https://github.com/react-native-community/discussions-and-proposals/discussions/802) and [for iOS](https://github.com/react-native-community/discussions-and-proposals/discussions/812).
+
+#### Android: JSIModule has been deleted
+
+The `com.facebook.react.bridge.JSIModule` ([source](https://github.com/facebook/react-native/blob/0.73-stable/packages/react-native/ReactAndroid/src/main/java/com/facebook/react/bridge/JSIModule.java)) was an API that we temporarily introduced to allow a Native Module to access JSI directly on Android.
+The accessors for this API [were deprecated in 0.74](https://reactnative.dev/blog/2024/04/22/release-0.74#other-breaking-changes), and we verified that there was no meaningful usage of this API in Open Source so we’re removing it in 0.75.
+You can use [Turbo Native Modules](https://github.com/reactwg/react-native-new-architecture/blob/main/docs/turbo-modules.md) instead as an alternative.
+
+#### Android: PopUp Menu moved to separate package
+
+In 0.74, [we moved the Android’s `PopUpMenu` to a separate package](https://reactnative.dev/blog/2024/04/22/release-0.74#other-breaking-changes) under the `@react-native` scope.
+In 0.75, we are removing the remaining methods that were in core:
+
+- `UIManagerModule.showPopupMenu()`
+- `UIManagerModule.dismissPopupMenu()`
+
+As alternative, please use the please use the `<PopupMenuAndroid />` component which lives in the [`@react-native/popup-menu-android`](https://www.npmjs.com/search?q=%40react-native%2Fpopup-menu-android) package.
+
+#### iOS: Finalized PushNotificationIOS deprecation work
+
+In 0.74, we [deprecated](https://reactnative.dev/blog/2024/04/22/release-0.74#api-changes-to-pushnotificationios-deprecated) some APIs from the `PushNotificationIOS` module.
+
+In 0.75, we’ve deleted these APIs to migrate off legacy representations of notifications metadata.
+
+The APIs that have been deleted are:
+
+```objc
+  + (void)didReceiveLocalNotification:(UILocalNotification *)notification;
+  + (void)didReceiveRemoteNotification:(NSDictionary *)notification;
+```
+
+Instead, use `didReceiveNotification:(UNNotification *)notification`.
+
+#### Community CLI: Removal of ram-bundle and profile-hermes commands
+
+We would like two significant removals coming from the Community CLI: the commands `ram-bundle` and `profile-hermes` have been removed.
```

Comment:
```suggestion
We would like to announce two significant removals coming from the Community CLI: the commands `ram-bundle` and `profile-hermes` have been removed.
```

Not sure if that is the intended wording, but something is definitely missing in this sentence.

Reply:
yup thank you